### ISSUE TITLE
adorn_ns() has the wrong ns after a tabyl is arrange()-d

### ISSUE THREAD

**Issue by sfirke** [sfirke/janitor#352]
```
> mtcars %>%
  tabyl(am, cyl) %>%
  arrange(desc(am)) %>%
  adorn_totals(c("row", "col")) %>%
  adorn_ns()

    am       4     6       8   Total
     1  8  (3) 3 (4)  2 (12) 13 (19)
     0  3  (8) 4 (3) 12  (2) 19 (13)
 Total 11 (11) 7 (7) 14 (14) 32 (32)
```

This is not a critical problem, you can feed custom Ns to the `adorn_ns()` call where you do the sort there too.  See the example here: https://stackoverflow.com/a/68776597/4470365.

But it has me wondering.  Many of the adorn_ functions call `as_tabyl` if the input is not a tabyl, maybe that `as_tabyl()` evaluation could recapture the `core` attribute of the tabyl in its post-sorted state?  Or is this opening a bigger can of worms.

Probably this issue will sit here forever but putting it here in case I'm on to something and realize it later :man_shrugging:  Prompted by #351

**Comment by sfirke** [comment]
Related to #407

**Comment by sfirke** [comment]
Well hey-hey, this was closed by #464
